### PR TITLE
HPCC-16516 Associate results with the activity that creates them

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday gavin.halliday@lexisnexis.com

### CODE REVIEW

**Comment by ghalliday** [comment]
@richardkchapman is this something worth including in 6.2?  Are the cassandra changes correct?

**Comment by richardkchapman** [comment]
Did you check/consider behaviour of Cassandra when connecting to an older Cassabnra database where the newly-added columns are not present?

**Comment by ghalliday** [comment]
no, I don't have a cassandra system set up

File: plugins/cassandra/cassandrawu.cpp [hunk 1]
```diff
@@ -1081,7 +1081,9 @@ static const ChildTableInfo wuGraphMetasTable =
         {"totalrowcount", "bigint", "totalRowCount", bigintColumnMapper},  /* This is the number of rows in value */ \
         {"schemaRaw", "blob", "SchemaRaw", blobColumnMapper},              \
         {"logicalName", "text", "logicalName", stringColumnMapper},        /* either this or value will be present once result status is "calculated" */ \
-        {"value", "blob", "Value", blobColumnMapper}
+        {"value", "blob", "Value", blobColumnMapper}, \
```
Comment (richardkchapman) [inline plugins/cassandra/cassandrawu.cpp]: And then add some code in the section titled:
                        // Add code here to create any columns that we need to to get from version "minor" to expected layout

**Comment by richardkchapman** [comment]
Replaced by #9360

**Pull request merged**